x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/Realms.java

```diff
@@ -178,6 +180,13 @@ public Realm realm(String name) {
                 }
                 internalTypes.add(type);
             }
+            if (KerberosRealmSettings.TYPE.equals(type)) {
+                if (isKerberosRealmConfigured) {
+                    throw new IllegalArgumentException(
+                            "multiple [" + type + "] realms are configured. [" + type + "] can only have one such realm configured");
```

Comment:
I know this is the same message as before - but could you extend it to include the names of the realms?